docset/winserver2022-ps/grouppolicy/Get-GPO.md

```diff
@@ -95,20 +110,24 @@ ComputerVersion  : AD Version: 0, SysVol Version: 0
 WmiFilter        :
 ```
 
-This command gets the GPO that has the ID (GUID) 331a09564-cd4a-4520-98fa-446a2af23b4b in the sales.contoso.com domain.
-If the domain of the user that is running the session (or, for startup and shutdown scripts, the computer) is different that sales.contoso.com, a trust must exist between the two domains.
+This command gets the GPO that has the ID (GUID) 331a09564-cd4a-4520-98fa-446a2af23b4b in the
+sales.contoso.com domain.
```

Comment:
In this case, we want to wrap both values in backticks because they represent specific values that are used in the command line:

```suggestion
This command gets the GPO that has the ID (GUID) `331a09564-cd4a-4520-98fa-446a2af23b4b` in the
`sales.contoso.com` domain.
```